### PR TITLE
Add dashboard screen

### PR DESCRIPTION
**Pull request by maricalucian**
This is work in progress.

### CODE REVIEW

File: src/components/account-card/account-card.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+                    </TextSmall>
+                </Text>
+                <TextSmall>{props.account.address} </TextSmall>
+            </View>
+            <Text>></Text>
+        </View>
+    );
+};
+
+export default AccountCard;
```
Comment (krisboit) [inline src/components/account-card/account-card.tsx]: I'm not a fan of default exports, we should agree on a format and use it for all components. With named exports autocomplete is working better...

File: src/components/conversion-card/conversion-card.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+                1
+            </Convert>
+            <TextSmall style={[change >= 0 ? styles.changeUp : styles.changeDown]}>
+                {change}
+            </TextSmall>
+        </View>
+    );
+};
+
+export default connect(
```
Comment (krisboit) [inline src/components/conversion-card/conversion-card.tsx]: As i said, i'm not a fan of default exports :), I'm thinking to do something like:
```
const ConversionCardComponent = () => {....}

export ConversionCard = connect(...)(ConversionCardComponent);
```
What do you think ?

File: src/components/account-card/account-card.tsx (hunk 1 above)
Comment (maricalucian) [reply src/components/account-card/account-card.tsx]: I`m all in for consistency. Default exporting is just a habit for components, we can use named exports.

File: src/components/conversion-card/conversion-card.tsx (hunk 2 above)
Comment (maricalucian) [reply src/components/conversion-card/conversion-card.tsx]: Yup, thats good going with named exports.

**Pull request merged**